src/index.js

```diff
@@ -17,7 +17,12 @@ const interval = setInterval(function () {
       if (isInternetDown) backUpAlert.play();
       console.log('Your internet connection is working!');
       const timeNow = new Date();
-      logsDiv.innerHTML += `<p class="success-log">${timeNow.toLocaleString()}: Internet is up.</p>`;
```

Comment:
I would suggest you to change this line from
`logsDiv.innerHTML += `<p ...` to 
`logsDiv.innerHTML = `<p></p> + logsDiv.innerHTML + `
so that the new logs will be displayed on top rather than going at the bottom